src/backend/marsha/core/utils/api_utils.py

```diff
@@ -49,3 +50,8 @@ def generate_hash(secret, message):
     return hmac.new(
         secret.encode("utf-8"), msg=message, digestmod=hashlib.sha256
     ).hexdigest()
+
+
+def generate_salted_hmac(secret, key):
+    """Generate a salted_hmac with secret and key"""
+    return salted_hmac(secret, key).hexdigest()[::2]
```

Comment:
use `sha256` as algo

Reply:
why so can you explain please

Reply:
`sha256` is the standard nowadays. It's stronger and more secure than `sha1`